systemd/teuthology-dispatcher@.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Teuthology Dispatcher
+
+Wants=ceph.target
+After=ceph.target
+
+[Service]
+Type=simple
+User=teuthworker
+ExecStart=/home/teuthworker/src/git.ceph.com_git_teuthology_main/virtualenv/bin/python3.8 \
```

Comment:
Just wondering why this refers to git.ceph.com, but not GitHub?
Also maybe there must be a parameter according to ansible or other scripts?

Reply:
git.ceph.com is the local mirror that virtually everything in teuthology uses, so it's the one most likely to be present and current.  don't know what parameters you mean exactly, but there is the 'instance' parameter so that one starts teuthology-dispatcher@smithi, for instance

Reply:
These files have several hardcoded elements that someone in a different setting will want to change; I'm not sure if there is a reasonable way to parametrize all of them, but I think it's fine to ship what are effectively templates that users can modify to suit their needs.